manifests/base/odh-dashboard-crd.yaml

```diff
@@ -92,14 +95,4 @@ spec:
                         type: string
                       lastSelectedSize:
                         type: string
-                      environmentVariables:
-                        type: array
-                        items:
-                          type: object
-                          properties:
-                            key:
-                              type: string
-                            value: 
-                              type: string
-                      secrets:
-                        type: string
```

Comment:
Same point as my last one... do not look to trim content as there is supporting code that needs it.